SCClassLibrary/Common/Collections/Scale.sc

```diff
@@ -1,17 +1,18 @@
 Scale {
-	var <degrees, <pitchesPerOctave, <tuning, <>name;
+	var <degrees, <stepsPerOctave, <tuning, <>name, <>baseFreq;
```

Comment:
renaming `pitchesPerOctave` to `stepsPerOctave` will cause all user-code to blow up, which uses keyword arguments. it may break existing pieces ...